src/components/field-label/field-label.js

```diff
@@ -51,11 +52,13 @@ export const FieldLabel = props => {
                 })}
               </Spacings.Inline>
             )}
-            {props.hint && <Text.Detail>{props.hint}</Text.Detail>}
+            {props.hint && (
+              <Text.Detail tone={props.tone}>{props.hint}</Text.Detail>
+            )}
           </Spacings.Inline>
         )}
         {props.description && (
-          <Text.Detail>
+          <Text.Detail tone={props.tone}>
```

Comment:
```suggestion
          <Text.Detail>
           <Text.Wrap tone={props.tone}>
          
```

Reply:
![Screen Shot 2019-10-31 at 2 25 14 PM](https://user-images.githubusercontent.com/2425013/67950650-4cbec300-fbea-11e9-94b9-9df6146e4c46.png)
We can tell from Percy that the tone is not being properly applied